### DISCUSSION TITLE
option to hide more of the UI  like twitter and chose ether twitter or Mastodon

### DISCUSSION THREAD

**Discussion by pederdm000** [tosdr/browser-extensions#82] [Ideas] [answered]
Having a option to hide more of the UI  like twitter and have a option to chose ether twitter or Mastodon or hide the option from the UI.

**Comment by JustinBack** [comment, marked as answer]
Added it to the Roadmap https://tosdr.atlassian.net/browse/ROAD-23 and moved it to jira from GitHub https://tosdr.atlassian.net/browse/BE-5
